include/rdma/fi_domain.h

```diff
@@ -115,6 +115,11 @@ struct fid_av {
  * Tracks registered memory regions, primarily for remote access,
  * but also for local access until we can remove that need.
  */
+
+#define FI_HMEM_HOST_ALLOC		(1ULL << 45)
+#define FI_HMEM_DEVICE_ONLY		(1ULL << 46)
+#define FI_HMEM_DATA_GDRCOPY_HANDLE	(1ULL << 59)
```

Comment:
Please move these to line 298 (where there's a comment regarding mr flags)

GDRCOPY_HANDLE should probably be something like GDR_MAP_HANDLE based on the comments in gdrapi.h:

https://github.com/NVIDIA/gdrcopy/blob/master/include/gdrapi.h